active-rfcs/0002-lifecycle.md

```diff
@@ -0,0 +1,68 @@
+- Feature Name: Lifecycle
+- Start Date: 2020-02-23
+- Reference Issues:
+- Implementation PR:
+
+# Summary
+
+We will develop a tichi plugin called lifecycle to track the status of each Issue or PR.
+
+# Motivation
+
+The TiDB community currently has a large number of stale issues, some of which may have been created several years ago. Some of these issues may not be discussed and tracked because the context is lost or the author never responds.
+
+The result is that these issues may never be fixed, but they will not be closed either. Because no one will pay attention to those issues that are already very old.
```

Comment:
please only consider PR firstly.

Reply:
This shouldn't be a big deal because the scan tool will scan against the GitHub Search API, so it doesn't matter if it's an issue or a PR.